modules/core/arrow-core-data/src/main/kotlin/arrow/core/Try.kt

```diff
@@ -43,9 +43,6 @@ sealed class Try<out A> : TryOf<A> {
         }
       }
 
-    @Deprecated(DeprecatedAmbiguity, ReplaceWith("raiseError(e)"))
-    fun raise(e: Throwable): Try<Nothing> = Failure(e)
-
```

Comment:
These looks like a merge error to me! 😱

Reply:
Yeah, I tried to rebase the branch to the latest version from the upstream fork